test/verify/check-system-realms

```diff
@@ -427,11 +435,13 @@ class CommonTests:
 
         # from sssd
         self.allow_journal_messages("alice is not allowed to run sudo on x0.  This incident will be reported.")
+        # occasional intermediate error during password auth
+        self.allow_journal_messages("cockpit-session: user account access failed: 4 alice: System error")
```

Comment:
The "4" is funny. But we can keep this for the time being, and replace with `.*` if/when it becomes a problem.